src/commands/hook/hook.js

```diff
@@ -2,9 +2,14 @@
 const HOOK: Object = {
   PERMISSIONS: 0o775,
   PATH: '/hooks/prepare-commit-msg',
-  CONTENTS:
-    '#!/bin/sh\n# gitmoji as a commit hook\n' +
-    'exec < /dev/tty\ngitmoji --hook $1\n'
+  CONTENTS: `#!/bin/sh
+# gitmoji as a commit hook
+if test -t 1 && ! grep -q -m 1 '^[^#]' $1; then
```

Comment:
Could you explain a little bit more this line ?

Seems a little bit tricky and hacky to my understand 😂

Reply:
I rephrased the comment that was below the condition.
The first condition means we are in a TTY (the hook has not been triggered from a program but likely from a user action), the second one means there is no comment already set.